backend/pack/pack.go

```diff
@@ -49,15 +52,41 @@ func (p *PackBuilder) Build(sourcePath string) error {
 
 	// Pull the pack image if not available
 	imageName := "buildpacksio/pack:latest"
-	_, _, err = p.dockerClient.ImageInspectWithRaw(p.ctx, imageName)
+	_, err = p.dockerClient.ImageInspect(p.ctx, imageName)
```

Comment:
was using a deprecated call apparently. I don't understand any of this code though. I guess we are just directly inspecting the error for "no such image". wonder how dagger handles this